src/obr/core/queries.py

```diff
@@ -238,3 +238,23 @@ def query_to_dataframe(
     if index:
         return ret.set_index(index).sort_index()
     return ret
+
+
+def filters_to_queries(filters: tuple[str]) -> list[Query]:
+    q: list[Query] = []
+    for filter in filters:
+        lhs, rhs = filter.split("=")
+        q.append(Query(key=lhs, value=rhs))
+    return q
+
+
+def filter_jobs_by_query(project, filter: tuple[str]) -> list[Job]:
```

Comment:
```suggestion
def filter_jobs(project, filter: tuple[str]) -> list[Job]:
```

Reply:
To separate filters and queries a bit better. We just use the query implementation to build a filter.